### PR TITLE
Improve rooms presentation

### PR DESCRIPTION
**Pull request by MarcoRLK**
# Porposed Changes

Response to https://github.com/MatheusRich/SIGS-GCES/issues/31. Creates another field to filter rooms.

# Type of Changes

What type of change this Pull Request brings to SIGS? Put an **x** in the boxes that apply

- [x] New backend feature or update.
- [x] New frontend feature or update.
- [ ] Bug fix.
- [ ] Another change (what was the change?).

# Checklist 

Put an **x** in the boxes that apply. If you're confused about any of the following topics, ask us. We're here to help you!
 
- [x] This Pull Request has a significant name.
- [x] The commits follow the [[commits policy]].
- [x] The build is okay (tests, code climate).
- [x] This Pull Request mentions a related issue.
- [x] The change was necessary to the progress of the project.

# Screenshots

image

### CODE REVIEW

File: SIGS/.rubocop.yml [hunk 1]
```diff
@@ -46,3 +46,7 @@ Metrics/BlockLength:
 Metrics/ModuleLength:
   Exclude:
     - 'spec/**/*.rb'
+
+Metrics/ClassLength:
+  Exclude:
+    - "app/controllers/rooms_controller.rb"
```
Comment (MatheusRich) [inline SIGS/.rubocop.yml]: Por que este arquivo foi removido do rubocop?

Comment (MatheusRich) [inline SIGS/.rubocop.yml]: Por que este arquivo foi removido do rubocop?

**Comment by MatheusRich** [comment]
Os departamentos estão aparecendo duplicados.

image

File: SIGS/app/controllers/rooms_controller.rb [hunk 2]
```diff
@@ -77,12 +94,8 @@ def json_of_categories_by_school_room
...
     allocations = Allocation.where(school_room_id: school_room_id)
     allocations.each do |allocation|
-      result.push [
-        allocation.start_time,
-        allocation.final_time,
-        allocation.day,
-        allocation.room.name
-      ]
+      result.push [allocation.start_time, allocation.final_time,
+                   allocation.day, allocation.room.name]
```
Comment (MatheusRich) [inline SIGS/app/controllers/rooms_controller.rb]: A identação ajuda a deixar o código mais legível. Mantenha-a.

File: SIGS/.rubocop.yml (hunk 1 above)
Comment (MarcoRLK) [reply SIGS/.rubocop.yml]: Exceção removida

File: SIGS/app/controllers/rooms_controller.rb (hunk 2 above)
Comment (MatheusRich) [reply SIGS/app/controllers/rooms_controller.rb]: Utilize as helpers do rails para remover simplificar as models.

Comment (MarcoRLK) [reply SIGS/app/controllers/rooms_controller.rb]: Certo, as mudanças foram realizadas. Alguns métodos saíram da controller e foram para uma helper criada para Rooms.

File: SIGS/app/controllers/rooms_controller.rb [hunk 3]
```diff
@@ -8,36 +8,23 @@ class RoomsController < ApplicationController
   def index
     @rooms = Room.all
     @buildings = Building.all
+    @department = Department.all
+    @user_department = find_user_department
+    puts(@user_department)
```
Comment (MatheusRich) [inline SIGS/app/controllers/rooms_controller.rb]: Remova esse `puts` e o PR está pronto para o merge.

Comment (MarcoRLK) [reply SIGS/app/controllers/rooms_controller.rb]: Removido

**Comment by MatheusRich** [comment]
Mergido! Boa refatoração @MarcoRLK.

**Pull request merged**